core/chains/evm/txmgr/broadcaster_test.go

```diff
@@ -90,6 +90,7 @@ func NewTestEthBroadcaster(
 	// Mark instance as test
 	ethBroadcaster.XXXTestDisableUnstartedTxAutoProcessing()
 	servicetest.Run(t, ethBroadcaster)
+	time.Sleep(time.Second) // let background initiate
```

Comment:
urgh, can we make this deterministic by waiting for a log message?

Reply:
I think waiting for `.Ready()` would be more robust, but it will require more changes and I didn't want this to snowball any more since it was incidental in this unrelated changeset. OK to follow up?
https://smartcontract-it.atlassian.net/browse/CRE-268